tests/utils/exec/rosacli/machinepool_service.go

```diff
@@ -162,6 +163,14 @@ func (m *machinepoolService) CreateMachinePool(
 	return
 }
 
+// Create MachinePool with no flags. Only used for validation in id:38775
+func (m *machinepoolService) CreateMachinePoolEmpty() (output bytes.Buffer, err error) {
```

Comment:
Not sure if this was the best practice for writing this part of the test case, please let me know if you have other ideas/suggestions.

Reply:
Let's not define such a function like this. Better to enhance function `CreateMachinePool` to make it support missing required flag.